packages/styles/src/components/tag.scss

```diff
@@ -0,0 +1,53 @@
+@forward './../variables/options';
+@forward './utilities';
+
+@use './../variables/components/tag' as tag;
+
+.tag:has(:not(pre)), // is only a temporary fix till I know how to do it wihtout causing future issues. wihtout this the Code sample would break currently.
+.tag-sm {
```

Comment:
Usually we define the size classes as modifiers. 
This means the `.tag` class adds all the default styles for a tag and the `.tag-sm` only modifies some properties to make the tag smaller. Then you would use `tag` for a default tag and `tag tag-sm` for a small tag. 

You can check how this is done for inputs and buttons for examples.